### DISCUSSION TITLE
Multi Modality

### DISCUSSION THREAD

**Discussion by aamir-gmail** [infiniflow/ragflow#2453] [General]
I see there i support for image2text in the supported model in ollama , I would resume they relate to llava:13b . I tried using them however the image I uploaded in the chat ended up in the knowledge base. My question is whether there is support for using Multi-modal models like llava 7 or 13 b

**Comment by KevinHuSh** [comment]
It supports image to text, but it does not support text to image.

**Comment by aamir-gmail** [comment]
when you say " It supports image to text", you mean multimodal models like
LLava
https://llava-vl.github.io/

On Mon, Sep 16, 2024 at 6:22 PM Kevin Hu ***@***.***> wrote:

> It supports image to text, but it does not support text to image.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/infiniflow/ragflow/discussions/2453#discussioncomment-10656723>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AJA2ECPGGASMCNVEURD6ZYTZW2IMRAVCNFSM6AAAAABOIUOIJSVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTANRVGY3TEMY>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Kind Regards


Aamir Mirza

**Reply by KevinHuSh** [reply]
Yes.

**Comment by mcammisa75** [comment]
Sorry if I reply to this quite old post...but, if I create a document with text and an image from the LLAVA training dataset and submit it to ragflow specifying to use that model (LLAVA7b) as Document Parser, should I expect the corresponding chunk contains a description of the image?
As a test I used this 
[Test.pdf](https://github.com/user-attachments/files/19507912/Test.pdf)
but the chunks contains just the test content. Is there anything I am missing?
Thanks
